### PR TITLE
[User] Search user

### PR DESCRIPTION
**Pull request by miguelbel00**
## Description
AS admin 
I WANT TO get all database users
SO I can get a list of users

## Evidence:
imagen

### CODE REVIEW

**Comment by miguelbel00** [comment]
Como Admin 
Quiero saber cuantos usuarios tengo
Para tener estadisticas

criterios de aceptacion 
GET /users - Deberia volver todos los usuarios, unicamente los campos de fisrtname, lastname,email y createdAt

imagen

**Review by IMatfres — CHANGES_REQUESTED** [review]
change the name file controller, for userSearchController, to keep all the same context, and in the file index route, your route should be /users and you have /users/

File: routes/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const express = require('express')
-const categoryRoutes = require('./categoryRoutes')
```
Comment (didierrevelo) [inline routes/index.js:2]: is lacki the category`s import

**Review by didierrevelo — CHANGES_REQUESTED** [review]
delete example files models

**Review by IMatfres — APPROVED** [review]
All Ok

**Review by didierrevelo — CHANGES_REQUESTED** [review]
all ok

**Review by didierrevelo — APPROVED** [review]

**Pull request merged**